packages/expo-app-auth/android/src/main/java/expo/modules/appauth/AppAuthModule.java

```diff
@@ -270,52 +292,53 @@ private void authWithConfiguration(
       final Map<String, String> parameters
   ) {
 
-    String responseType = ResponseTypeValues.CODE;
-    if (parameters.containsKey("response_type")) {
-      String responseTypeString = parameters.remove("response_type");
-      if ("token".equals(responseTypeString)) {
-        responseType = ResponseTypeValues.TOKEN;
-      } else if ("id_token".equals(responseTypeString)) {
-        responseType = ResponseTypeValues.ID_TOKEN;
-      }
-    }
+    Activity activity;
+
     try {
-      AuthorizationRequest.Builder authReqBuilder = new AuthorizationRequest.Builder(serviceConfig, clientId, responseType, Uri.parse(redirectUrl));
+      activity = getCurrentActivity();
+    } catch (Exception error) {
+      mAuthTask.reject(error);
+      return;
+    }
 
-      if (scopes != null) {
-        String scopesString = Serialization.scopesToString(scopes);
-        if (scopesString != null) {
-          authReqBuilder.setScope(scopesString);
-        }
+    AuthorizationRequest.Builder authReqBuilder = new AuthorizationRequest.Builder(serviceConfig, clientId, ResponseTypeValues.CODE, Uri.parse(redirectUrl));
```

Comment:
looks like you removed the `responseType` parsing and the try-catch block i added in https://github.com/expo/expo/pull/6716 -- was that intentional?

Reply:
oh, I see now that this PR is pretty old, which explains why that stuff is missing. If you could rebase/merge with master so this PR doesn't unintentionally revert #6716, that would be great!